apis/chatbotapi/chatbotapi.yaml

```diff
@@ -0,0 +1,219 @@
+openapi: 3.0.1
+info:
+  title: Chatbot
+  description: 'These are the sunbird chatbot APIs for interacting with TARA as assistance.'
+  termsOfService: https://github.com/project-sunbird/sunbird-commons/blob/master/LICENSE
+  contact:
+    email: info@sunbird.org
+  version: "1.0"
+  
+externalDocs:
+ description: "Testcases"
+ url: "https://www.getpostman.com/collections/aad34d927df640d90727"
+paths:
+  /bot:
```

Comment:
this should be /chat/bot @JagadishPujari , please make sure you add /chat to all the paths below as well.